.drone.yml

```diff
@@ -137,7 +137,7 @@ steps:
     image: nextcloudci/php7.3:php7.3-5
     environment:
       APP_NAME: spreed
-      CORE_BRANCH: master
+      CORE_BRANCH: stable21
       GUESTS_BRANCH: master
```

Comment:
It looks like `GUESTS_BRANCH` needs to be changed too :-)

Reply:
Not needed, because guests app doesn't have such branches.

See https://github.com/nextcloud/guests/pull/543

Reply:
:facepalm: :+1: